frontend/client/src/app/api/model-utils.ts

```diff
@@ -0,0 +1,47 @@
+import {
+  global_palette_purple_400 as criticalColor,
+  global_danger_color_100 as importantColor,
+  global_info_color_100 as lowColor,
+  global_warning_color_100 as moderateColor,
+} from "@patternfly/react-tokens";
+
+import { Severity } from "./models";
+import { ProgressProps } from "@patternfly/react-core";
+
+type ListType = {
+  [key in Severity]: {
+    shieldIconColor: { name: string; value: string; var: string };
+    progressProps: Pick<ProgressProps, "variant">;
+  };
+};
+
+export const severityList: ListType = {
+  low: {
+    shieldIconColor: lowColor,
+    progressProps: { variant: undefined },
+  },
+  moderate: {
+    shieldIconColor: moderateColor,
+    progressProps: { variant: "warning" },
+  },
+  important: {
+    shieldIconColor: importantColor,
+    progressProps: { variant: "danger" },
+  },
+  critical: {
+    shieldIconColor: criticalColor,
+    progressProps: { variant: "danger" },
+  },
+};
+
+export const severityFromNumber = (score: number): Severity => {
```

Comment:
Would it be useful to get words converted from numbers from the backend? Would future localizations be easier/harder? Should l10n stuff be only in the front-end, leaving the backend simpler? 

I don't know nor do I have an opinion, just questions.

Reply:
`Would it be useful to get words converted from numbers from the backend?`

Definitely it would be easier. The current Trustification returns a number. But you're right, this is thing to enhance and leave it to the backend to interpret/convert scores (numbers).

So I'll leave it to the backend .

It seems I will need to redefine my DTOs (mock DTOs, not linked to the backend yet). https://github.com/carlosthe19916/trustify/blob/advisory-list/frontend/client/src/app/api/models.ts#L73

`Would future localizations be easier/harder`
localyzation should not be affected. It won't make it easier nor harder, just not affect.

Reply:
> mock DTOs

+1